### PR TITLE
Game logic: Milestone 3

### PR DESCRIPTION
**Pull request by rindrajosia**
In this branch we:
Add real logic for your game.
main.rb file is responsible for user interaction

### CODE REVIEW

**Review by serdg0 — CHANGES_REQUESTED** [review]
Hi @rindrajosia & @Stricks1 !
Very good job! Overall, this project is awesome! You are **VERY** close to finish it! :racing_car: :racing_car: 

We only require to change some minor issues in order to get your project approved:

- [ ]  Classes should be in the lib folder.

- [ ] There is 1 inline comment.

- [ ] The draw message is always being printed.

After adressing those minor bugs submit anoter CRR.

**Happy Coding!** :computer: :nerd_face:
**_Sergio Diaz_** (edited)

File: bin/main [hunk 1]
```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts "Hello world!"
+require '../lib/board.rb'
+require '../lib/player.rb'
+require '../lib/winning.rb'
+require '../lib/playagain.rb'
```
Comment (suretrust) [inline bin/main]: Use `require_relative` for files within the folder structure. Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @rindrajosia, 👋 

### Great job implementing the suggested changes. 🙌 However, I couldn't run your app. 👇 

- [ ] See my comments on your code to locate the bug.

Screenshot 2020-05-15 at 10 39 15

**Kindly submit for another CRR once these issues have been fixed.** _Happy coding!_ 🎉

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team!
### You have a conflict in the main.rb file, solve and submit for another code review. Good luck.
conflict

**Comment by rindrajosia** [comment]
> Hello Team!
> ### You have a conflict in the main.rb file, solve and submit for another code review. Good luck.
> 
> conflict

Hello 
thanks for the review however we don't face this merge conflict in our computeur.
Anyway, we have updated our main.rb to try to resolve this merge conflict even if we don't see it when we make a pull request.
conflict

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Hi Team,
### Good work, you did everything required previously :+1:  Approved.
Just before you merge this project consider a few more changes:
* Why do you need a separate file for the `play_again` method? Please move this method to `main` file and delete `playagain.rb` file.
* Add a validation that will not allow the player to have a blank name.

Then go ahead and merge into development.
_Goodluck in your next milestone_ *Happy coding*

**Pull request merged**